app/components/bali/carousel/index.js

```diff
@@ -0,0 +1,46 @@
+import { Controller } from '@hotwired/stimulus'
+
+export class CarouselController extends Controller {
+  static values = {
+    type: { type: String, default: 'carousel' },
+    index: { type: Number, default: 0 },
```

Comment:
Para ser consistente con el API de Glide

```suggestion
    startAt: { type: Number, default: 0 },
```